jenkins/image_building/build-image.sh

```diff
@@ -13,11 +13,26 @@ source "${current_dir}/upload-ci-image.sh"
 source "${current_dir}/upload-node-image.sh"
 
 # Disable needrestart interactive mode
-sudo sed -i "s/^#\$nrconf{restart} = 'i';/\$nrconf{restart} = 'a';/" /etc/needrestart/needrestart.conf > /dev/null
+sudo sed -i "s/^#\$nrconf{restart} = 'i';/\$nrconf{restart} = 'a';/" /etc/needrestart/needrestart.conf > /dev/null || true
 
 sudo apt-get update
 sudo apt-get install python3-pip qemu qemu-kvm -y
 sudo pip3 install diskimage-builder python-openstackclient
+# TODO(Sunnatillo): When newer version than 3.32.0 of disk-image builder released remove changes
+# done by this commit. 
+sudo -H pip install virtualenv
+
+# sudo pip3 install diskimage-builder python-openstackclient
+mkdir "${current_dir}/dib"
+cd "${current_dir}/dib"
+virtualenv env
+# shellcheck disable=SC1091
+source env/bin/activate
+git clone https://opendev.org/openstack/diskimage-builder
```

Comment:
First off great that you fixed this but some thoughts that popped in my mind, not 100% necessary
 
Maybe checkout the specific commit you verified as there is some concerns regarding stability and security when just taking master without knowing what has changed there since last time?

The other option we could do here is simply build the package upload as a tarball to artifactory and then pip install directly from there as pip allows installations from urls. Then it would be quite minimal temporary change at least but, I guess that is mostly cosmetics

Reply:
Thank you for the review. Valid point. I changed it to specific commit.